sdk/tables/azure-data-tables/samples/async_samples/sample_copy_client_async.py

```diff
@@ -0,0 +1,108 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_copy_table_async.py
+
+DESCRIPTION:
+    These samples demonstrate how to copy tables between table and blob storage.
+
+USAGE:
+    python sample_copy_table_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_STORAGE_CONNECTION_STRING - the connection string to your storage account
+    2) TABLES_STORAGE_ENDPOINT_SUFFIX - the Table service account URL suffix
+    3) TABLES_STORAGE_ACCOUNT_NAME - the name of the storage account
+    4) TABLES_PRIMARY_STORAGE_ACCOUNT_KEY - the storage account access key
+"""
+
+
+from datetime import datetime
+import os
+import json
+import asyncio
+from azure.storage.blob.aio import BlobClient, BlobServiceClient
+from azure.data.tables.aio import TableClient
+from dotenv import find_dotenv, load_dotenv
+
+
+class CopyTableSamples(object):
+    def __init__(self):
+        load_dotenv(find_dotenv())
+        self.access_key = os.getenv("TABLES_PRIMARY_STORAGE_ACCOUNT_KEY")
+        self.endpoint_suffix = os.getenv("TABLES_STORAGE_ENDPOINT_SUFFIX")
+        self.account_name = os.getenv("TABLES_STORAGE_ACCOUNT_NAME")
+        self.endpoint = "{}.table.{}".format(self.account_name, self.endpoint_suffix)
+        self.table_connection_string = "DefaultEndpointsProtocol=https;AccountName={};AccountKey={};EndpointSuffix={}".format(
+            self.account_name, self.access_key, self.endpoint_suffix
+        )
+        self.blob_connection_string = os.getenv("AZURE_STORAGE_CONNECTION_STRING")
```

Comment:
Maybe it's possible to use the same storage connection string for both Tables and Blobs?

Reply:
I don't think so, for example, it's using different connection strings when running set-up resource scripts.

Reply:
It should be possible, as most Storage accounts support both blobs and tables in the same account. However we can keep them separate in case someone wants to combine Storage blobs with Cosmos tables etc